packages/app/app/ui/page.tsx

```diff
@@ -26,6 +26,7 @@ import {
   Toast,
   Severity,
 } from "@/ui";
+import { FREQUENCY_OPTIONS } from "../../models/stack";
```

Comment:
can we import like this? 

```suggestion
import { FREQUENCY_OPTIONS } from "@/models/stack";
```